src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -82,7 +82,7 @@ public static SystemEnforcementMode GetLockdownPolicy(string path, SafeHandle ha
             // At this point, LockdownPolicy = Audit or Allowed.
             // If there was a WLDP policy, but WLDP didn't block it,
             // then it was explicitly allowed. Therefore, return the result for the file.
-            SystemEnforcementMode systemWldpPolicy = s_cachedWldpSystemPolicy.GetValueOrDefault(SystemEnforcementMode.None);
+            SystemEnforcementMode systemWldpPolicy = s_cachedWldpSystemPolicy ?? SystemEnforcementMode.None;
```

Comment:
Can you revert the changes to this file and make them in a separate PR?  They will require more review.